src/components/SUE/index.ts

```diff
@@ -1 +1,4 @@
+export * from './SUEAddress';
+export * from './SUECategory';
 export * from './SUEMapView';
+export * from './SUEStatus';
```

Comment:
Sue...?

Reply:
Do you mean you would prefer "Sue..." over "SUE..." or don't use Sue at all in the naming? @donni106

Reply:
He prefers to spell it Sue instead of SUE. Sue is a new module and if Sue is not written, conflicts will occur in the code. So we need Sue somewhere.

https://github.com/smart-village-solutions/smart-village-app-app/pull/961/commits/8aaeb9c9211bbc5c1fa7404380e9a8006e782ad4

Reply:
yep